docs/companion.md

```diff
@@ -67,6 +67,42 @@ files to arrive at Transloadit servers, much like Uppy.
 
 :::
 
+To do so each provider plugin must be configured with Transloadit’s Companion
+URLs:
+
+```js
+import { COMPANION_URL, COMPANION_ALLOWED_HOSTS } from '@uppy/transloadit';
+import Dropbox from '@uppy/dropbox';
+
+uppy.use(Dropbox, {
+	companionUrl: COMPANION_URL,
```

Comment:
I think we should also line out what actual values can be inserted for `COMPANION_URL` 

People often don't get this.

Reply:
> I think we should also line out what actual values can be inserted for `COMPANION_URL`

The idea is that folks do not insert actual values, but import it from `@uppy/transloadit`

Reply:
Yes, I know that. Customers are sometimes overwhelmed and do not get it managed (somehow).